WarSpecialists/Assets/Player/Scripts/Maria/MariaAbility.cs

```diff
@@ -112,7 +125,10 @@ void Ability1()
                         Vector3 directionToTarget = transform.position - enemy.gameObject.transform.position;
                         float angel = Vector3.Angle(transform.forward, directionToTarget);
                         if (Mathf.Abs(angel) > 90 && Mathf.Abs(angel) < 270 && directionToTarget.magnitude <= abilityRange)
-                            Debug.Log("Ability1 to enemy!");
+                            if (enemy.name == "Red_Minion")
+                            {
+                                enemy.gameObject.GetComponent<Minion>().TakeDamage(damage);
+                            }
```

Comment:
Why enemy.name? The player has a component called Targetable that has a variable called Team, you should compare with enemy's team, if it's different than yours, you do damage.  it should be: 
```suggestion
                            if (enemy.team != gameObject.getComponent<Targetable>().team)
                            {
                                enemy.gameObject.GetComponent<Minion>().TakeDamage(damage);
                            }
```